setup.md

```diff
@@ -27,12 +27,28 @@ Provision a V100 using this key id
 ibmcloud sl vs create \
     --datacenter=lon04 \
```

Comment:
nit: should we change this to wdc07  too?

Reply:
updated